JPS_Ontology/ontology/ontodevice/OntoDevice.csv

```diff
@@ -0,0 +1,58 @@
+hasHeight,Object Property,,,Device,http://www.ontology-of-units-of-measure.org/resource/om-2/Height,,A relationship between a device and its height.,https://www.theworldavatar.com/kg/ontodevice/,has height
+hasWidth,Object Property,,,Device,http://www.ontology-of-units-of-measure.org/resource/om-2/Width,,A relationship between a device and its width.,https://www.theworldavatar.com/kg/ontodevice/,has width
+hasPrice,Object Property,,,Device,http://www.ontology-of-units-of-measure.org/resource/om-2/AmountOfMoney,,A relationship between a device and its price.,https://www.theworldavatar.com/kg/ontodevice/,has price
+hasMass,Object Property,,,Device,http://www.ontology-of-units-of-measure.org/resource/om-2/Mass,,A relationship between a device and its mass.,https://www.theworldavatar.com/kg/ontodevice/,has mass
+hasSystemCapability,Object Property,,,Device,http://www.w3.org/ns/ssn/systems/SystemCapability,,A relationship between a device and its system capability.,https://www.theworldavatar.com/kg/ontodevice/,has system capability
+hasOperatingRange,Object Property,,,Device,http://www.w3.org/ns/ssn/systems/OperatingRange,,A relationship between a device and its operating range.,https://www.theworldavatar.com/kg/ontodevice/,has operating range
+hasSurvivalRange,Object Property,,,Device,http://www.w3.org/ns/ssn/systems/SurvivalRange,,A relationship between a device and its survival range.,https://www.theworldavatar.com/kg/ontodevice/,has survival range
+hasSetpoint,Object Property,,,Device,Setpoint,,A relationship between a device and a setpoint.,https://www.theworldavatar.com/kg/ontodevice/,has setpoint
+hasQuantity,Object Property,,,Setpoint,http://www.ontology-of-units-of-measure.org/resource/om-2/Quantity,,A relationship between a setpoint or operating property and the quantity it represents.,https://www.theworldavatar.com/kg/ontodevice/,has quantity
+hasQuantity,Object Property,,,http://www.w3.org/ns/ssn/systems/OperatingProperty,http://www.ontology-of-units-of-measure.org/resource/om-2/Quantity,,A relationship between a setpoint or operating property and the quantity it represents.,https://www.theworldavatar.com/kg/ontodevice/,has quantity
+https://www.theworldavatar.com/kg/ontotimeseries/hasTimeSeries,Object Property,,,https://saref.etsi.org/core/State,https://www.theworldavatar.com/kg/ontotimeseries/TimeSeries,,,https://www.theworldavatar.com/kg/ontodevice/,has timeseries
+https://www.theworldavatar.com/kg/ontotimeseries/hasTimeSeries,Object Property,,,http://www.ontology-of-units-of-measure.org/resource/om-2/Measure,https://www.theworldavatar.com/kg/ontotimeseries/TimeSeries,,,https://www.theworldavatar.com/kg/ontodevice/,has timeseries
+https://www.theworldavatar.com/kg/ontotimeseries/hasTimeSeries,Object Property,,,http://www.opengis.net/ont/sf#Point,https://www.theworldavatar.com/kg/ontotimeseries/TimeSeries,,,https://www.theworldavatar.com/kg/ontodevice/,has timeseries
```

Comment:
hasTimeSeries already has owl:Thing as domain, so the above three definitions are unnecessary. Please remove.